superset/models/core.py

```diff
@@ -222,6 +223,17 @@ def unique_name(self) -> str:
     def url_object(self) -> URL:
         return make_url(self.sqlalchemy_uri_decrypted)
 
+    @property
+    def parameters(self) -> Optional[Dict[str, Any]]:
+        # Build parameters if db_engine_spec is a subclass of BasicParametersMixin
+        parameters = {"engine": self.backend}
```

Comment:
so for sqlalchemy forms, we're only going to return engine? Maybe that should be the only required type then, it looks like.

Reply:
Yea good catch i'll make sure to update the type on the types file